github/PullRequest.py

```diff
@@ -727,6 +727,26 @@ def is_merged(self) -> bool:
         status, headers, data = self._requester.requestJson("GET", f"{self.url}/merge")
         return status == 204
 
+    def enable_automerge(self, merge_method: Opt[str] = "MERGE") -> None:
+        """
+        :calls: `POST /graphql <https://docs.github.com/en/graphql>`_ with a mutation to enable pull request auto merge
+        <https://docs.github.com/en/graphql/reference/mutations#enablepullrequestautomerge>
+        """
+        assert is_optional(merge_method, str), merge_method
+        mutation = "mutation EnablePullRequestAutoMerge($input: EnablePullRequestAutoMergeInput!) { enablePullRequestAutoMerge(input: $input) { clientMutationId } }"
+
+        # Define the variables
+        variables = {"input": {"pullRequestId": self.raw_data["node_id"], "mergeMethod": merge_method}}
+
+        post_parameters = {"query": mutation, "variables": variables}
+
+        # Make the request
+        responseHeaders, data = self._requester.requestJsonAndCheck(
+            "POST", "https://api.github.com/graphql", input=post_parameters
+        )
+        if "errors" in data:
+            raise self._requester.createException(400, responseHeaders, data)
+
     def merge(
```

Comment:
While implementing `enable_automerge`, method `disable_automerge` could be implemented as well.